SU-SWS/decanter
# D8CORE-6970 | @jdwjdwjdw | Update local footer headlines

# READY FOR REVIEW

# Summary
- [D8CORE-6970](https://stanfordits.atlassian.net/browse/D8CORE-6970): A11y| Footer: H2 displays differently based on location
  - Per related ticket, updating all `h2`'s in the local footer to be 18px, with `h3` - `h5` updated to `16px`.

# Needed By (Date)
- When convenient

# Urgency
- How critical is this PR?

# Steps to Test

1. Checkout branch
2. Navigate to `/admin/config/system/local-footer` and fill out all of the fields to match the [Amptesting site](https://amptesting.sites.stanford.edu/admin/config/system/local-footer). Save page and navigate to a page you can view the local footer.
3. Confirm that all of the `h2`'s in the local footer (including the `Content Blocks` section) are the same size (18px) and h3-h5's are 16px.
4. Review code 🦕 

# Associated Issues and/or People
- [D8CORE-6970](https://stanfordits.atlassian.net/browse/D8CORE-6970): A11y| Footer: H2 displays differently based on location

[D8CORE-6970]: https://stanfordits.atlassian.net/browse/D8CORE-6970?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[D8CORE-6970]: https://stanfordits.atlassian.net/browse/D8CORE-6970?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
